jax/lax_linalg.py

```diff
@@ -40,6 +40,13 @@ def qr(x, full_matrices=True):
   q, r = qr_p.bind(x, full_matrices=full_matrices)
   return q, r
 
+def svd(x, full_matrices=True, compute_uv=True):
+  s, u, v = qr_p.bind(x, full_matrices=full_matrices, compute_uv=compute_uv)
```

Comment:
Typo: this should probably be `svd_p.bind`.

Reply:
Oh, my bad. I'll fix this in the next commit.